internal/feedback/feedback.go

```diff
@@ -115,3 +164,14 @@ func GetLockingFeedback(version, revision, depType, projectPath string) string {
 	}
 	return fmt.Sprintf("Locking in %s (%s) for %s %s", version, revision, depType, projectPath)
 }
+
+func trimSHA(revision string) string {
```

Comment:
Added this convenience method, which we could leverage above at https://github.com/golang/dep/pull/1475/files#diff-ca9bf3aa3bbb62e361e500352f67474bR152

Reply:
👍